### PR TITLE
Publish Solaris 11 package generation

### PR DESCRIPTION
**Pull request by DFolchA**
Hello team,

This PR includes the Solaris 11 package generation in the wazuh-packages repository.

Best regards,
Daniel Folch

### CODE REVIEW

File: solaris/packer/package_generation/vagrant/Vagrantfile [hunk 1]
```diff
@@ -108,8 +108,10 @@ Vagrant.configure("2") do |config|
     os.each do |key, value|
         config.vm.define "#{key}" do |node|
           node.vm.box = "#{value['box']}"
-          if key == "solaris10"
-            node.vm.box_url = 'https://packages-dev.wazuh.com/utils/vagrant/solaris/10/solaris10.box'
+          if key == "solaris11"
+            node.vm.box_url = 'https://s3-us-west-1.amazonaws.com/packages-dev.wazuh.com/utils/vagrant/solaris/11/i386/solaris11.3.box'
```
Comment (alberpilot) [inline solaris/packer/package_generation/vagrant/Vagrantfile]: Please remove `s3-us-west-1.amazonaws.com` from URL

File: solaris/packer/package_generation/vagrant/Vagrantfile [hunk 2]
```diff
@@ -108,8 +108,10 @@ Vagrant.configure("2") do |config|
...
+          elsif key == "solaris10"
+            node.vm.box_url = 'https://s3-us-west-1.amazonaws.com/packages-dev.wazuh.com/utils/vagrant/solaris/10/solaris10.box'
```
Comment (alberpilot) [inline solaris/packer/package_generation/vagrant/Vagrantfile]: Same here

File: solaris/solaris10/README.md [hunk 3]
```diff
@@ -8,20 +8,20 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
...
+    $ git clone https://github.com/wazuh/wazuh-packages && cd wazuh-packages/solaris/solaris10
     ```
 
-2. Execute the `generate_wazuh_packages.sh` script to build the package. There are multiple parameters to select which package is going to be built, its architecture, etc. Here you can see all the different parameters:
+2. Execute the `generate_wazuh_packages.sh` script to build the package. Here you can see all the different parameters:
     ```shellsession
     # ./generate_wazuh_packages.sh -h
 
-    Usage: ./generate_wazuh_rpm.sh [OPTIONS]
+    Usage: ./generate_wazuh_packages.sh [OPTIONS]
```
Comment (alberpilot) [inline solaris/solaris10/README.md]: Please change the `Build deb packages.` part.

File: solaris/solaris10/README.md [hunk 4]
```diff
@@ -8,20 +8,20 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
 To build a Wazuh package you need to install the following tools:
   - `git`: `pkgutil -i git`.
 
-## Building RPM packages
+## Building Solaris11 packages
```
Comment (alberpilot) [inline solaris/solaris10/README.md]: Solaris 10 or 11?

File: solaris/solaris11/README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+In this repository, you can find the necessary tools to build a Wazuh package for Solaris 11.
+
+## Tools needed to build the package
+
+To build a Wazuh package you need to install the following tools:
+  - `git`: `pkgutil -i git`.
+
+## Building Solaris11 packages
+
+To build a Solaris 10 package, you need to download this repository and use the `generate_wazuh_packages.sh` script. This script will download the source code from the [wazuh/wazuh](https://github.com/wazuh/wazuh) repository and generate a `.p5p` package.
```
Comment (alberpilot) [inline solaris/solaris11/README.md]: Solaris 10 or 11?

File: solaris/solaris11/uninstall.sh [hunk 6]
```diff
@@ -0,0 +1,22 @@
+#/bin/sh
+
+## Stop and remove application
+sudo /var/ossec/bin/ossec-control stop
+sudo rm -r /var/ossec*
+sudo rm /etc/ossec-init.conf
+
+## stop and unload dispatcher
+#sudo /bin/launchctl unload /Library/LaunchDaemons/com.wazuh.agent.plist
```
Comment (alberpilot) [inline solaris/solaris11/uninstall.sh]: Unnecessary line

File: solaris/packer/package_generation/vagrant/Vagrantfile (hunk 1 above)
Comment (DFolchA) [reply solaris/packer/package_generation/vagrant/Vagrantfile]: Done in https://github.com/wazuh/wazuh-packages/pull/191/commits/49424d4fb30ffd2950a5e8cb0d1fac8c69fdf893

File: solaris/packer/package_generation/vagrant/Vagrantfile (hunk 2 above)
Comment (DFolchA) [reply solaris/packer/package_generation/vagrant/Vagrantfile]: Done in https://github.com/wazuh/wazuh-packages/pull/191/commits/49424d4fb30ffd2950a5e8cb0d1fac8c69fdf893

File: solaris/solaris10/README.md (hunk 3 above)
Comment (DFolchA) [reply solaris/solaris10/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/191/commits/49424d4fb30ffd2950a5e8cb0d1fac8c69fdf893

File: solaris/solaris10/README.md (hunk 4 above)
Comment (DFolchA) [reply solaris/solaris10/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/191/commits/49424d4fb30ffd2950a5e8cb0d1fac8c69fdf893

File: solaris/solaris11/uninstall.sh (hunk 6 above)
Comment (DFolchA) [reply solaris/solaris11/uninstall.sh]: Done in https://github.com/wazuh/wazuh-packages/pull/191/commits/49424d4fb30ffd2950a5e8cb0d1fac8c69fdf893

File: solaris/solaris11/README.md (hunk 5 above)
Comment (DFolchA) [reply solaris/solaris11/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/191/commits/49424d4fb30ffd2950a5e8cb0d1fac8c69fdf893

**Pull request merged**